lib/bindings/http/emitter_structured.js

```diff
@@ -1,30 +1,23 @@
 var axios = require("axios");
 
 const Constants = require("./constants.js");
+const defaults = {};
+defaults[Constants.HEADERS] = {};
+defaults[Constants.HEADERS][Constants.HEADER_CONTENT_TYPE] = Constants.DEFAULT_CE_CONTENT_TYPE;
 
 function StructuredHTTPEmitter(configuration){
-  this.config = JSON.parse(JSON.stringify(configuration));
-
-  this.config[Constants.HEADERS] =
-    (!this.config[Constants.HEADERS]
-      ? {}
-      : this.config[Constants.HEADERS]);
-
-  if(!this.config[Constants.HEADERS][Constants.HEADER_CONTENT_TYPE]){
-    this.config[Constants.HEADERS][Constants.HEADER_CONTENT_TYPE] =
-      Constants.DEFAULT_CE_CONTENT_TYPE;
-  }
+  this.config = Object.assign({}, defaults, configuration);
```

Comment:
Why are we using `Object.assign` when we can just use the native JavaScript object spread operator?

```js
this.config = {...defaults, ...configuration};
```

Reply:
The commit happened before we dropped support for early versions of Node.js so this would not have passed CI. It's a reasonable change for master now that we've dropped 6 and 8.

Reply:
I'm not sure of the CI system yet but Node versions < 10 were end of life Dec 31, 2019. Completely understandable, but we should only use non-deprecated versions of Node in our CI.

Reply:
@grant yes, I am aware. That's why I updated CI last month to only use 10 & 12. https://github.com/cloudevents/sdk-javascript/commit/aa2cef6e3cb3a399afbea24006cc83d8d70af920#diff-354f30a63fb0907d4ad57269548329e3

Reply:
@lance I've created this #119 but I don't think a new PR is necessary maybe if you and the others agree with that practice you can update your PR instead wdyt ?

Reply:
@helio-frota I think submitting a separate PR for that is fine.